### PR TITLE
fix: get fields with comma

### LINKED ISSUE #59: Failure to properly retrieve items when there is a comma inside the field
**Describe the bug**

When listing items that have commas, it breaks and shows only the "first" part of the item. 

**To Reproduce**
1. Create an item in a test vault with a comma. This might even be a password.
2. Get the item and additional items

**Expected behavior**
The item, quoted.

**Screenshots**
Nope.

**Additional context**

This happens because there is a `split(",")` usage in the library. I'll provide a patch.

### PR DESCRIPTION
**Pull request by fzipi360**
Fixes #59

### CODE REVIEW

**Comment by dtpryce** [comment]
I need to dig into this one a little bit but will try to test out this week

File: onepassword/client.py [hunk 1]
```diff
@@ -432,11 +432,13 @@ def get_item(uuid: str | bytes, fields: str | bytes | list | None = None):
...
-                "op item get {} --fields label={}".format(uuid, ",label=".join(fields)),
-                single=False
-            ).rstrip('\n')
-            item = dict(zip(fields, items.split(",")))
+            items = json.loads(read_bash_return(
+                "op item get {} --format=json --fields label={}".format(uuid, ",label=".join(fields)),
+                single=False))
+            item = {}
+            for i in items:
+                label = i["label"]
```
Comment (dtpryce) [inline onepassword/client.py:440]: Ran a test using a login item that had a password with a comma in it. Performed:
```
from onepassword import OnePassword

op = OnePassword()
uuid = op.get_uuid("test login")
password = op.get_item(uuid, fields=["password"])

```
And got following error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "REDACTED/1password-client/onepassword/client.py", line 440, in get_item
    label = i["label"]
TypeError: string indices must be integers
```
And from quick scan you need to make sure you are using python 3.10 or higher here. Dictionaries are iterated using this syntax:

```
for k, v in dictionary.items():
   print(k, v)
```

Hope that helps with your next commit :D

**Review by dtpryce — CHANGES_REQUESTED** [review]

File: onepassword/client.py (hunk 1 above)
Comment (fzipi360) [reply onepassword/client.py:440]: Ah, good catch! Will update to support older versions.

Comment (fzipi360) [reply onepassword/client.py]: Used `.items()` in f3ee92a6652a80f9ad780d859c2d03e754618d90. Do you think we can mock a test for this, or do we need to use manual testing?

**Pull request closed without merging**